doc/user/content/sql/create-source/json-kinesis.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "CREATE SOURCE: JSON over Kinesis"
-description: "Learn how to connect Materialize to an Avro-formatted Kafka topic"
+title: "[BETA] CREATE SOURCE: JSON over Kinesis"
```

Comment:
Placing this in the page title is very confusing because it makes it seem like everything else is not in beta, when AFAIK it is!